README.md

```diff
@@ -55,9 +55,17 @@ export interface ExampleSchema {
 
 ## Usage
 
+Check [sample app](sample/foo-typescript) for more details.
+
 ```js
 import {compileFromFile} from 'json-schema-to-typescript'
 fs.writeFileSync('foo.d.ts', await compileFromFile('foo.json'))
+
+async function generate() {
+  fs.writeFileSync('dist/foo.d.ts', await compileFromFile('src/foo.json'))
+}
```

Comment:
The assumption is that the containing function is async. I made it the top level fn here for brevity, as most users already know that async can't be top level.

Your complete example in /example should suffice I think.